ci-scripts/dev-sandbox/run.sh

```
@@ -29,6 +31,14 @@ collect_counts() {
     done
 }
 
+collect_heap_profile() {
+    id=${1:-default}
+    if [ "$COLLECT_HEAP_PROFILE" == "true" ]; then
+        pod=$(oc -n rhdh-operator get pods -l control-plane=controller-manager -o json | jq -rc '.items[] | select(.metadata.name | startswith("rhdh-operator")).metadata.name')
+        oc -n rhdh-operator exec pod/$pod -c manager -- bash -c "curl -SsL localhost:6060/debug/pprof/heap | base64" | base64 -d >"$ARTIFACT_DIR/dev-sandbox/rhdh-operator-heap.$id.pprof"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)